main.py

```diff
@@ -0,0 +1,73 @@
+from fastapi import FastAPI, Depends, HTTPException
+from sqlalchemy.orm import Session
+
+import crud
+import schemas
+from database import SessionLocal
+
+app = FastAPI()
+
+
+def get_db() -> Session:
+    db = SessionLocal()
+
+    try:
+        yield db
+    finally:
+        db.close()
+
+
+@app.get("/authors/", response_model=list[schemas.Author])
+def read_authors(
+    skip: int | None = 0,
```

Comment:
The type hint `int | None` is not compatible with Python versions below 3.10. If you are using an earlier version, consider using `Optional[int]` from the `typing` module instead.